### DISCUSSION TITLE
Discover LocalSend on different subnet

### DISCUSSION THREAD

**Discussion by wowbufer** [localsend/localsend#1254] [Discussion (English, International)]
Hello,

well I have a Local Send PC on one subnet (192.168.5.0/24) and while WiFi devices are on subnet (192.168.6.0/24).

I can ping anything from anywhere and i can send files via Local Send if i write manual IP address, but discovery is not working, I have enabled IGMP snooping on Access point and on router. Still discovery dosen't work.

anybody have advice or solution what to do? or maybe somebody already did configuration like i have.

Mikrotik is router, ubiquiti is access point.

Cheers :)

**Comment by Tienisto** [comment]
You can add favorites if it doesn't work.

**Reply by wowbufer** [reply]
I have common new people sending me files at my print studio (at least 5/people a day), so it's a bit annoying on long run to do it every day, if you get me

**Reply by Tienisto** [reply]
Yeah, it is indeed annoying.
Currently, LocalSend (1) broadcasts a UDP package to scan its network and also (2) sends a HTTP request to all devices in all connected subnets (which works more reliable than UDP/multicast).
Since your devices live in different subnets, the second method seem to fail while the first doesn't work at all.
Unfortunately, I don't have a fix for this either...

**Reply by wowbufer** [reply]
Thank you for your time :)
I have workaround with buying NIC and put it in PC for recieving files, that NIC will have 192.168.6.0/24 and everything will be good :)

LocalSend is awesome app, just my usecase is a bit different from others.

Thank you one more time @Tienisto

**Comment by Bouni** [comment]
I have the same problem, PC is on one subnet, wifi devices on another. It would be nice if I could add favorites by IP without having to discover them first. That would be a nice workaround.

**Reply by wowbufer** [reply]
Kinda, im setting up LocalSend for one print studio in my town, so there comes at least 5-10new people every day. I created one open SSID with firewall rules that can access only LocalSend pc over that SSID and on that way we are eliminating sending files over email for printing. 

So in my scenario even that favorite IP is not much of help since constantly new people are coming to print studio.

Cheers :)

**Comment by gh2023-aesun** [comment]
I have a similar use-case.  I'd like to send file(s) to a device on a different subnet, that my device has network access to.  My device will never be on the same subnet as it's at a separate location.

Currently, trying to add a favorite for a device on a different subnet fails, even though I can access the device over the network.

Is there any way to fix adding any device accessible by IP, as a favorite?

**Comment by broomwalker** [comment]
Also found this to be an issue between this and no linux headless, I’m finding that this software may not be for me.
